build.gradle

```diff
@@ -96,10 +96,8 @@ dependencies {
     compile 'com.opencsv:opencsv:3.4'
     compile 'com.google.guava:guava:18.0'
     compile 'com.github.samtools:htsjdk:'+ htsjdkVersion
-    // Using the shaded version to avoid conflicts between its protobuf dependency
-    // and that of Hadoop/Spark (either the one we reference explicitly, or the one
-    // provided by dataproc).
-    compile 'com.google.cloud:gcloud-java-nio:0.2.8:shaded'
+    // Don't use the shaded version, since it includes un-relocated netty, which causes conflicts with Spark
+    compile 'com.google.cloud:gcloud-java-nio:0.2.8'
```

Comment:
[PR 1327](https://github.com/GoogleCloudPlatform/google-cloud-java/pull/1327) is merged, so this CL will be unblocked by the first gcloud release that includes it.